### PR TITLE
Update CocoaPods (1.6.0.rc.1), SwiftLint (0.30.1), Kanna (4.0.3) and StencilSwiftKit (2.7.2)

### PR DESCRIPTION
**Pull request by djbe**
Refs #582.

### CODE REVIEW

File: .swiftlint.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ opt_in_rules:
  - convenience_type
  - discouraged_optional_boolean
  - discouraged_optional_collection
+ - duplicate_imports
```
Comment (ikesyo) [inline .swiftlint.yml]: FYI This and `unused_setter_value` are not opt-in rules.

```
$ swiftlint rules | grep -e opt-in -e duplicate_imports -e unused_setter_value
| identifier                               | opt-in | correctable | enabled in your config | kind        | analyzer | configuration |
| duplicate_imports                        | no     | no          | yes                    | idiomatic   | no       | warning       |
| unused_setter_value                      | no     | no          | yes                    | lint        | no       | warning       |
```

Comment (djbe) [reply .swiftlint.yml]: Huh, didn't know that. Man they should note that (consistently) in the changelog.

**Comment by djbe** [comment]
Found some more rules that we enable but aren't opt-in, and 2 old ones that I forgot to enable.

**Comment by djbe** [comment]
I've got a 6.1.0 release branch ready on top of this one, so after we merge this, we can release it really quickly.

**Pull request merged**